### ISSUE TITLE
mloginfo --clients is missing old nodejs clients

### ISSUE THREAD

**Issue by niccottrell** [rueckstiess/mtools#808]
### Expected behavior

Log lines like the following should show up in the final summary output:
```
2020-07-02T13:37:12.208+0000 I NETWORK  [conn202099] received client metadata from 192.168.181.44:36650 conn202099: { driver: { name: "nodejs", version: "2.2.36" }, os: { type: "Linux", name: "linux", architecture: "x64", version: "3.10.0-1062.12.1.el7.x86_64" }, platform: "Node.js v8.16.0, LE, mongodb-core: 2.1.20" }
```

### Actual/current behavior
These lines are not included in the result

### Steps to reproduce the actual/current behavior
run `mloginfo --clients` on a file with that log line

### Environment

| Software         | Version
| ---------------- | -------
| mtools           |  1.6.3 
| Python           | 3.7.6
| MongoDB server   | 3.6
| Operating system | macOS

**Comment by niccottrell** [comment]
Looks like valid JSON to me:

```js
{
  driver: {
    name: "nodejs",
    version: "2.2.36"
  },
  os: {
    type: "Linux",
    name: "linux",
    architecture: "x64",
    version: "3.10.0-1062.12.1.el7.x86_64"
  },
  platform: "Node.js v8.16.0, LE, mongodb-core: 2.1.20"
}
```

**Comment by stennie** [comment]
Hi @niccottrell,

`mloginfo --clients` converts the raw output into valid JSON which includes quoting the field names. The regex wasn't expecting commas in the `platform` value.

I committed a fix for mtools 1.6.4.

Cheers,
Stennie
